### DISCUSSION TITLE
CIE 1994 colour difference implementation might generate NaN values.

### DISCUSSION THREAD

**Discussion by pfk-beta** [colour-science/colour#683] [General]
I do not have certain pixel for reproducing error, but according to http://brucelindbloom.com/index.html?Eqn_DeltaE_CIE94.html footnotes it is possible. I would like to report it, and solve pull request.

**Comment by thomasmansencal** [comment]
Hi @pfk-beta!

Thanks for reporting this! I think it would be great to identify which cases are generating NaNs first.  I'll continue the discussion in #539! See you there :)

Cheers,

Thomas
